### PR TITLE
Fix typo in Tide js and display action target.

### PR DESCRIPTION
**Pull request by cjwagner**
screenshot from 2017-12-13 12 34 28

Previously the `Passing` column was always empty.  And the `State` column did not specify the target.

### CODE REVIEW

File: prow/cmd/deck/static/tidescript.js [hunk 1]
```diff
@@ -71,22 +71,37 @@ function createLinkCell(text, url, title) {
...
     if (prs) {
         for (var i = 0; i < prs.length; i++) {
             var a = document.createElement("a");
             a.href = "https://github.com/" + pool.Org + "/" + pool.Repo + "/pull/" + prs[i].Number;
             a.appendChild(document.createTextNode("#" + prs[i].Number));
-            c.appendChild(a);
-            c.appendChild(document.createTextNode(" "));
+            elem.appendChild(a);
+            if (i+1 < prs.length) {
+                elem.appendChild(document.createTextNode(" "));
```
Comment (BenTheElder) [inline prow/cmd/deck/static/tidescript.js]: why exactly are we appending a " " node?

Comment (rmmh) [reply prow/cmd/deck/static/tidescript.js]: Yeah, add a comment here. The logic is tricky.

Comment (cjwagner) [reply prow/cmd/deck/static/tidescript.js]: Its a space between the PR number links.

Comment (BenTheElder) [reply prow/cmd/deck/static/tidescript.js]: I suppose not. Maybe add a comment breifly mentioning it what way this adds PRs to the element? "adds a list of space separated PR links". The method name is terse for no comments.

Comment (cjwagner) [reply prow/cmd/deck/static/tidescript.js]: I added some comments.

**Pull request merged**